templates/strings-dot-syntax-swift3.stencil

```diff
@@ -10,12 +10,14 @@ import Foundation
 // swiftlint:disable nesting
 // swiftlint:disable variable_name
 // swiftlint:disable valid_docs
+{% macro parametersBlock params %}{% for type in params.types %}_ p{{forloop.counter}}: {{type}}{% if not forloop.last %}, {% endif %}{% endfor %}{% endmacro %}
```

Comment:
Ah, right. Might be worth nitpicking it in a separate PR btw (that we could merge either before or after the Great Split and the next major version, depending on our agenda)